### PR TITLE
Search criteria and Controller class.

### PR DESCRIPTION
**Pull request by manu863018**
Review code please.

### CODE REVIEW

File: src/main/java/com/fundation/search/controller/Controller.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package com.fundation.search.controller;
+
```
Comment (PanDeBatalla94) [inline src/main/java/com/fundation/search/controller/Controller.java]: Copyright is missing

File: src/main/java/com/fundation/search/controller/Controller.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        //Validator validator = new Validator();
+        /**
+         * if (validator.validateName(name) && validator.validatePath(name)) {
+         *     setSearchCriteria(name, path);
+         * }
+         */
+
+    }
+
+    public void setSearchCriteria(String name, String path) {
```
Comment (PanDeBatalla94) [inline src/main/java/com/fundation/search/controller/Controller.java]: comments are missing

File: src/main/java/com/fundation/search/controller/SearchCriteria.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package com.fundation.search.controller;
+
```
Comment (PanDeBatalla94) [inline src/main/java/com/fundation/search/controller/SearchCriteria.java]: missing copyright and comments

File: src/main/java/com/fundation/search/controller/Controller.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    /**
+     * Method to initialize view.
+     */
+    public void initViewModel() {
+        //view = new View();
+        //model = new Model();
+    }
+
+    public void getFieldsFromView() {
+        //name = JTextFieldName.getText();
```
Comment (JimmyRomero) [inline src/main/java/com/fundation/search/controller/Controller.java]: Remove and Avoid uploading commented-out lines of code

**Pull request merged**